kubernetes/mysql-service.yaml

```
@@ -1,14 +1,15 @@
   labels:
     app: mysql
 spec:
+  clusterIP: None
   selector:
-    app: mysql
+    app: mysql 
   ports:
-  - protocol: TCP
-    port: 3306
-    targetPort: 3306
+   - protocol: TCP
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 3

(indentation)